debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -313,6 +313,22 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
                     + "In '" + SnapshotIsolationMode.READ_UNCOMMITTED.getValue()
                     + "' mode neither table nor row-level locks are acquired, but connector does not guarantee snapshot consistency.");
 
+    public static final Field STREAMING_DATABASE_CALLBACKS = Field.createInternal("streaming.database.callbacks")
```

Comment:
Unless it's still necessary, please use a single parameter. Otherwise, once this is contributed upstream, we may have to migrate the connectors from the old to the new configuration.